cmd/snap/cmd_asserts.go

```diff
@@ -0,0 +1,85 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/ubuntu-core/snappy/asserts"
+	"github.com/ubuntu-core/snappy/client"
+	"github.com/ubuntu-core/snappy/i18n"
+	"github.com/ubuntu-core/snappy/logger"
+)
+
+type assertsOptions struct {
+	AssertTypeName string   `positional-arg-name:"assertion-type" description:"assertion type name" required:"true"`
+	HeaderFilters  []string `positional-arg-name:"header-filters" description:"header=value" required:"false"`
+}
+
+type cmdAsserts struct {
+	assertsOptions `positional-args:"true" required:"true"`
+}
+
+var (
+	shortAssertsHelp = i18n.G("Asserts searches the system for assertions of the given type")
```

Comment:
By the way, what is the convention for the short help?  I'm surprised to see the command name uppercased.